odk/odk.py

```diff
@@ -317,7 +317,7 @@ class ReportConfig(JsonSchemaMixin):
     release_reports : bool = False
     """ If true, release reports are added as assets to the release (top level directory, reports directory)"""
     
-    custom_sparql_checks : Optional[List[str]] = field(default_factory=lambda: ['equivalent-classes', 'owldef-self-reference'])
+    custom_sparql_checks : Optional[List[str]] = field(default_factory=lambda: ['equivalent-classes', 'owldef-self-reference', 'taxon-range', 'iri-range'])
```

Comment:
dont add 'taxon-range' to default better

Reply:
Ok. Do I include both `present_in_taxon` and `never_in_taxon` in the iri-range?

Reply:
Yeah. And rdfs:seeAlso, dcterms:contributor and whatever else you can think of

Reply:
It doesn't work `isIRI` check with `anyURI` type that are the cases you included. I'll only include the three properties `in_subset`, `present_in_taxon` and `never_in_taxon`.

Reply:
Yeah I forgot. you can check "isIRI(?x)" or (DATATYPE(?x)=xsd:anyUri). What do you think?

Reply:
Sure, I was also thinking about it. Found many violations using `xsd:string`.